### DISCUSSION TITLE
[Feature Request] 能否提供个admin模板

### DISCUSSION THREAD

**Discussion by w6et** [element-plus/element-plus#6117] [Q&A]
### Existing Component
否

### Component Name
admin pro

### Description
目前各大前端框架，都会提供一套中后台解决方案，https://beta-pro.ant.design/index-cn、https://devui.design/admin，
不知道官方是否有此计划

**Comment by Lupeiwen0** [comment]
肯定会有admin的模板的 但是目前他们应该顾不上，刚发布稳定版本 还有好多 问题没有修复。后面有时间应该会搞，不过你可以搜搜其他人的 admin 模板，看看有没有适合你的

**Reply by linjialiang** [reply]
@Lupeiwen0 构建文档会报警告，无法找到 window 这个顶层， 这个有空应该优化下

**Comment by pany-ang** [comment]
官方可能没有暂时没有这个计划，你可以试试三方的，比如https://github.com/un-pany/v3-admin

**Comment by emojiiii** [comment]
有需要可以先暂时使用 https://github.com/xiaoxian521/vue-pure-admin.

**Reply by xiaoxian521** [reply]
https://github.com/element-plus/rfcs/issues/1

**Comment by Cheerwhy** [comment]
自己撸一个，现在element夜间模式都还没做完

**Comment by linjialiang** [comment]
> 有需要可以先暂时使用 https://github.com/xiaoxian521/vue-pure-admin.

我正在使用，3.0版本了，已经稳定，后面就是修bug了，功能不会做重大调整

**Comment by notlaughingzzm** [comment]
我正在使用，3.0版本了，已经稳定，后面就是修bug了，功能不会做重大调整 https://github.com/vbenjs/vue-vben-admin

**Comment by tolking** [comment]
一千个人心中就有一千个哈姆雷特，管理模版我觉得还是自己组织更加合适
